kura/org.eclipse.kura.wire.component.provider/src/main/java/org/eclipse/kura/internal/wire/common/DbServiceHelper.java

```diff
@@ -76,7 +76,6 @@ public static DbServiceHelper getInstance(final DbService dbService) {
      *             if argument is null
      */
     public void close(final Connection conn) {
-        requireNonNull(conn, message.connectionNonNull());
```

Comment:
The check if Connection is null is already performed in the Db implementation. So, having the check here is not needed. Furthermore, we should try to keep consistency and this is not the case if we throw an exception when trying to close the connection, while we accept to close a null statement.